recipes/qbs/all/conanfile.py

```diff
@@ -0,0 +1,125 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class QbsConan(ConanFile):
+    name = "qbs"
+    description = "The Qbs build system"
+    topics = ("conan", "qbs", "build", "automation")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://qbs.io"
+    license = "LGPL-2.1-only", "LGPL-3.0-only", "Nokia-Qt-exception-1.1"
+    settings = "arch", "build_type", "compiler", "os"
+    no_copy_source=True
+    exports = ["patches/*.patch"]
+    build_policy = "missing"
+
+    build_requires = "qt/5.15.2"
+
+    def build_requirements(self):
+        if tools.os_info.is_windows and self.settings.compiler == "Visual Studio":
+            self.build_requires("jom/1.1.3")
+
+    def configure(self):
+        if self.settings.os not in ["Windows", "Linux", "Macos"]:
+            raise ConanInvalidConfiguration("The OS ({}) is not supported by {}.".format(self.settings.os, self.name))
+        if self.settings.arch not in ["x86_64"]:
+            raise ConanInvalidConfiguration("The arch ({}) is not supported by {}.".format(self.settings.arch, self.name))
+
+        self.options["qt"].gui = False
+        self.options["qt"].openssl = False
+        self.options["qt"].qtscript = True
+        self.options["qt"].shared = False
+        self.options["qt"].widgets = False
+        self.options["qt"].with_odbc = False
+        self.options["qt"].with_pq = False
+        self.options["qt"].with_sqlite3 = False
+        self.options["qt"].with_zstd = False
+        for dep, options in self._data["options"].items():
+            for key, value in options.items():
+                setattr(self.options[dep], key, value)
+
+        minimal_cpp_standard = "11"
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, minimal_cpp_standard)
+
+        minimal_version = {
+            "gcc": "5.0",
+            "clang": "6.0",
+            "apple-clang": "11",
+            "Visual Studio": "16",
+        }
```

Comment:
This list seems way above the C++11 requirement

Reply:
What would be your suggestion? We are bound by https://doc.qt.io/qt-5/supported-platforms.html